### ISSUE TITLE
multiline-comment-style fix produces invalid code

### ISSUE THREAD

**Issue by epmatsw** [eslint/eslint#9461]
**Tell us about your environment**

* **ESLint Version:** 4.9.0
* **Node Version:** 8.6.0
* **npm Version:** 5.5.1

**What parser (default, Babel-ESLint, etc.) are you using?** default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
    "parserOptions": {
        "ecmaVersion": 8,
        "sourceType": "script",
        "ecmaFeatures": {}
    },
    "rules": {
        "multiline-comment-style": 2
    },
    "env": {}
}
```

</details>

**What did you do? Please include the actual source code causing the issue.**

```js
//blah
////blah
//blah
```

**What did you expect to happen?**
It should produce a valid comment after fixing.
**What actually happened? Please include the actual, raw output from ESLint.**
It produces:

```javascript
/*
 *blah
 *//blah
 *blah
 */
```

In both eslint and a browser, this produces a parsing error: `Parsing error: Unterminated regular expression (Fatal)` and `Exception: SyntaxError: unterminated regular expression literal` respectively.

**Comment by Aladdin-ADD** [comment]
thanks for reporting, I can repro it!
